src/sonic-py-common/sonic_py_common/multi_asic.py

```diff
@@ -241,15 +236,15 @@ def get_all_namespaces():
     if is_multi_asic():
         for asic in range(num_asics):
             namespace = "{}{}".format(ASIC_NAME_PREFIX, asic)
-            if namespace not in config_db_handle:
-                config_db_handle[namespace] =  connect_config_db_for_ns(namespace)
-            config_db = config_db_handle[namespace]
-
+            config_db =  connect_config_db_for_ns(namespace)
             metadata = config_db.get_table('DEVICE_METADATA')
             if metadata['localhost']['sub_role'] == FRONTEND_ASIC_SUB_ROLE:
                 front_ns.append(namespace)
             elif metadata['localhost']['sub_role'] == BACKEND_ASIC_SUB_ROLE:
                 back_ns.append(namespace)
+        if config_db:
+            # Ensure the client socket is closed
+            config_db.close('CONFIG_DB')
```

Comment:
I see build failure at this close call
  File "/usr/local/lib/python3.9/dist-packages/sonic_py_common/multi_asic.py", line 218, in get_all_namespaces
    config_db.close('CONFIG_DB')
  File "/usr/local/lib/python3.9/dist-packages/swsssdk/dbconnector.py", line 271, in close
    self.dbintf.close(db_name)
  File "/usr/local/lib/python3.9/dist-packages/swsssdk/interface.py", line 216, in close
    self.redis_clients[db_name].connection_pool.disconnect()
AttributeError: 'SwssSyncClient' object has no attribute 'connection_pool'